test/mock_gps_tests/MockGPSNucTest.cpp

```diff
@@ -0,0 +1,279 @@
+// Copyright 2017 UBC Sailbot
+
+// COPIED FROM projects/nuc_eth_listener/main.cpp
```

Comment:
Thanks for this comment. Also gonna add doxygen style comment and Henry as an author